src/main/java/com/seailz/javadiscordwrapper/DiscordJv.java

```diff
@@ -425,4 +428,31 @@ public Cache<Channel> getChannelCache() {
     public void addIntent(@NotNull Intent intent) {
         intents.add(intent);
     }
+
+    private void registerCommand(Command command) {
+        DiscordRequest commandReq = new DiscordRequest(
+                command.compile(),
+                new HashMap<>(),
+                URLS.POST.COMMANDS.GLOBAL_COMMANDS.replace("{application.id}", getSelfInfo().id()),
+                this,
+                URLS.BASE_URL,
+                RequestMethod.POST);
+        commandReq.invoke();
+    }
+
+    /**
+     * Clears all the global commands for this app. Cannot be undone.
+     * @apiNote This currently does not work, don't use.
```

Comment:
wdym does not work

Reply:
you need to send an empty `new JSONObject("[]");`

Reply:
when i did this, it requires JSON strings to begin with "{"

Reply:
Try {[]}

Reply:
won't work, you can't directly nest json elements

Reply:
when attempting to send {"body":[]}, i'm getting a 411 LENGTH_REQUIRED error. trying to solve this within `DiscordRequest.java` with `con.setRequestProperty("Content-Length", String.valueOf(body.length()));` does not work

Reply:
You could try adding another private method to the DiscordRequest.java class that accepts a string, and then adding a public method for a JSONArray that just runs the string method, and then just make the JSONObject method run the string method. Does that make sense?